web/app/components/document/sidebar/related-resources/list.ts

```diff
@@ -71,6 +71,17 @@ export default class DocumentSidebarRelatedResourcesListComponent extends Compon
       return;
     }
 
+    if (insertedSprites.length === 1 && removedSprites.length === 1) {
+      /**
+       * For unknown reasons, Ember Animated sees an updated sprite
+       * as both inserted and removed, causing unwanted animations.
+       * This check prevents that from happening.
+       *
+       * FIXME: Investigate and fix this.
+       */
+      return;
+    }
+
```

Comment:
A rather disappointing side effect of the refactor: Recently edited sprites are no longer interpreted by Ember Animated as `kept` but rather `removed` _and_ `inserted`.

I have some theories that I'll investigate soon.